index.html

```diff
@@ -8,6 +8,22 @@
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <header>
+        <h1 class="nav-brand">My Logo</h1>
+        <img src="./images/icons/Union.svg" alt="Menu" />
+    </header>
```

Comment:
- [ ] Great job utilizing semantic tags, However, it is one of the project requirements to use an `<a></a>` tag for the logo kindly fix that.

Reply:
👍🏽 Anchor tag implemented.